ui/src/testing/factories/bootresource.ts

```diff
@@ -0,0 +1,58 @@
+import { define, extend } from "cooky-cutter";
+
+import { model } from "./model";
+
+import type {
+  BootResource,
+  BootResourceEventError,
+  BootResourceOtherImage,
+  BootResourceStatuses,
+  BootResourceUbuntu,
+  BootResourceUbuntuCoreImage,
+} from "app/store/bootresource/types";
+import { BootResourceAction } from "app/store/bootresource/types";
+import type { Model } from "app/store/types/model";
+
+export const bootResource = extend<Model, BootResource>(model, {
+  rtype: 0,
+  name: "ubuntu/bionic",
+  title: "18.04 LTS",
+  arch: "amd64",
+  size: "650.4 MB",
+  complete: false,
+  status: "Waiting for rack controller(s) to sync",
+  icon: "waiting",
+  downloading: false,
+  numberOfNodes: 0,
+  lastUpdate: "Tue, 08 Jun. 2021 02:12:47",
+});
+
+export const bootResourceUbuntu = define<BootResourceUbuntu>({
+  arches: () => [],
```

Comment:
should we have factories for each of `arches`, `releases` and `sources` as well?

Reply:
Yeah we should, I'll add them.